draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -242,11 +242,42 @@ At a high level, the Verifier
 
 The detailed algorithm is described in (#verifier_verification).
 
-# Data Formats {#data_formats}
+# SD-JWT Data Formats {#data_formats}
 
-This section defines data formats for SD-JWT including the Issuer-signed JWT content, Disclosures, and Key Binding JWT.
+An SD-JWT is composed of the following:
+
+* an Issuer-signed JWT,
+* zero or more Disclosures, and
+* optionally a Key Binding JWT.
+
+The individual parts will be explained in the following subsections.
+
+The serialized format for the SD-JWT is the concatenation of each part delineated with a single tilde ('~') character as follows:
+
+```
+<Issuer-signed JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>~<optional KB-JWT>
```

Comment:
```suggestion
<Issuer-signed JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>~<optional KB-JWT>

```

adding a newline so (hopefully) the horizontal scrollbar in the HTML output doesn't cover up the content